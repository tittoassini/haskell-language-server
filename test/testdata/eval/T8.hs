-- Semantic and Lexical errors are reported
module T8 where

-- >>> noFunctionWithThisName

-- >>> "a" + "bc"

-- >>> "

-- >>> div 3 0
