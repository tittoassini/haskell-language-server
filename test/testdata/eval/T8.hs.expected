-- Semantic and Lexical errors are reported
module T8 where

-- >>> div 3 0
-- divide by zero
